src/main/zip/plugin.xml

```diff
@@ -1471,10 +1611,16 @@
                      default-value="${p?:resource/iib.executionGroup}"
                      description="Comma or new line separated list of Execution Groups to Deploy to."/>
       </property>
-      <property name="fullDeploy">
-        <property-ui type="checkBox"
-                     label="Full Deploy"
-                     description="If true empties Execution Group before Deploy."/>
+      <property name="deployType">
+        <property-ui type="selectBox"
+                     label="Deployment Type"
+                     default-value="${p?:iib.deployType}"
+                     description="Specify either an incremental or full deployment. You may 
+                        specify the 'Parameter' option and set the 'iib.deployType' property 
+                        to either 'incremental' or 'full' at any level in the property hierarchy."/>
+        <value label="Full">full</value>
+        <value label="Incremental">incremental</value>
+        <value label="Parameter">${p?:iib.deployType}</value>
```

Comment:
The label should be `${p?:iib.deployType}` too. The user will need to know what parameter to set.

Reply:
Why is the default the parameter? Shouldn't it be full? That's what we've had to date, right?